auto-cpufreq.conf-example

```diff
@@ -22,6 +22,17 @@ energy_performance_preference = performance
 # turbo boost setting. possible values: always, auto, never
 turbo = auto
 
+
+# this is for ignoring controlers and other connected devices battery from effecting
```

Comment:
spelling correction: controllers and affecting

Reply:
Thx!

Reply:
np

Reply:
fixed!